index.html

```diff
@@ -20,7 +20,17 @@
         </ul>
       </nav>
     </header>
-    <main></main>
+    <main>
+      <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
+      <p>En pleins centre ville ou en pleine nature</p>
+      <form>
+        <!--Ajout d'une barre de recherche-->
+        <p>
+          <input type="search" name="BSearch" id="BSearch" placeholder="Marseille,France"/>
```

Comment:
Please put "search" on id and name. please add a space before France. Type must be a type exist on HTML.